pom.xml

```diff
@@ -56,6 +56,7 @@
         <module>extensions/server/kyuubi-server-plugin</module>
         <module>extensions/spark/kyuubi-extension-spark-jdbc-dialect</module>
         <module>extensions/spark/kyuubi-spark-authz</module>
+        <module>extensions/spark/kyuubi-spark-authz-bundle</module>
```

Comment:
can we use the term "shaded" instead of "bundle", as we already have `kyuubi-hive-jdbc-shaded`

Reply:
Both are fine for me, and I prefer the `bundle` as it is a external client-side jar.

Reply:
`kyuubi-hive-jdbc-shaded` does too, I prefer use `shaded` for consistent

Reply:
Ok